sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosContainer.java

```diff
@@ -632,6 +632,10 @@ private CosmosBatchResponse blockBatchResponse(Mono<CosmosBatchResponse> batchRe
         return  this.blockDeleteItemResponse(asyncContainer.deleteItem(itemId, partitionKey, options));
     }
 
+    public CosmosItemResponse<Object> deleteAllItemsByPartitionKeyAsync(PartitionKey partitionKey, CosmosItemRequestOptions options) {
```

Comment:
`deleteAllItemsByPartitionKeyAsync` or `deleteAllItemsByPartitionKey`?

please check other APIs in this class, follow the same naming pattern

Reply:
Replaced with deleteAllItemsByPartitionKey